File: benchmarks/bench_fib.py
import time, sys

def fib(n):
    if n <= 1: return n
    return fib(n-1) + fib(n-2)

t0 = time.perf_counter()

result = 0

for r in range(5):
    result = fib(40)

ms = (time.perf_counter() - t0) * 1000

print(result)
print(f'COMPUTE_MS={ms:.3f}', file=sys.stderr)

File: benchmarks/bench_fib.js
const { performance } = require('perf_hooks');
function fib(n) {
    if (n <= 1) return n;
    return fib(n-1) + fib(n-2);
}
const t0 = performance.now();
let result = 0;
for (let r = 0; r < 5; r++) result = fib(40);
const ms = performance.now() - t0;
console.log(result);
process.stderr.write(`COMPUTE_MS=${ms.toFixed(3)}\n`);
